web/src/assets/css/common.css

```diff
@@ -253,7 +253,7 @@
 .el-button--text:active,
 .el-button--text:focus,
 .el-button--text:hover {
-    color: #1f83e7 !important;;
+    color: #1f83e7 !important;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of !important](https://app.codacy.com/gh/WeBankFinTech/WeBASE-Front/pullRequest?prid=5240357)